Manual/Classes/InstanceSynth.lean

```diff
@@ -81,6 +83,41 @@ Search may succeed, fail, or get stuck; a stuck search may occur when an unknown
 Stuck searches may be re-invoked when the elaborator has discovered one of the previously-unknown implicit arguments.
 If this does not occur, stuck searches become failures.
 
+:::example "Tracing instance search"
+
+The {option}`trace.Meta.synthInstance` option provides a powerful tool for
+
+```lean (name := trace)
+set_option pp.explicit true in
+set_option trace.Meta.synthInstance true in
+#synth Nonempty (Nat ⊕ Empty)
+```
+```leanOutput trace
+[Meta.synthInstance] ✅️ Nonempty (Sum Nat Empty)
+```
+
+The online version of the manual, the Lean InfoView, and the live lean editor (available via the "Live ↪" button below) show a clickable "▶" symbol which will iteratively allow a more thorough investigation of how Lean succeeds, or fails, at type class instance synthesis.
```

Comment:
Nitpick: I'm not sure that the adverb "iteratively" meaningfully applies to "allow" here. What about:
```suggestion
The online version of this manual, the Lean InfoView, and the live lean editor (available via the "Live ↪" button below) include a clickable "▶" symbol in the traces.
Clicking it expands that branch of the trace.
This allows an iterative and thorough investigation of how Lean succeeds, or fails, at type class instance synthesis.
```